### DISCUSSION TITLE
Having multiple Message Note

### DISCUSSION THREAD

**Discussion by LancerTony** [mido/mido#431] [Q&A] [answered]
Using the `mido.msg.note`, I can get a note that is played by the user. I want to identify two notes and for example print a string that the user has played two notes(or more), but I can't do that.

Prematurely I thought of this, but I know it doesn't work
`import mido
 for msg in in_port:
    print(msg.note)
        if(msg.note == 48 and msg.note == 52):
           `
Is there a better way of doing this?

**Comment by olemb** [comment, marked as answer]
This will never be true:

```python
if msg.note == 48 and msg.note == 52:
```

since `msg.note` can never have two different values at the same time.

If you want to detect if the user played note 48 or note 52 you could do something like this:

```python
for msg in in_port:
    if msg.type == 'note_on':
        if msg.note == 48 or msg.note == 52:
            print(msg.note)
```

Note that I'm checking the message type. A note is made up of two messages: a note on and a note off (for the start and end of the note). There could also be other message types that you'd want to ignore in this case, such as control changes caused by sustain pedals or other control.

I hope this helps!
